contributing/development/debugging/vulkan/vulkan_validation_layers.rst

```diff
@@ -3,10 +3,87 @@
 Validation layers
 =================
 
-Validation layers enable developers to verify their application's correct use
-of the Vulkan API.
+Validation layers enable developers to verify their application's correct use of
+the Vulkan API. Validation layers can be enabled in both debug and release
+builds, including in exported projects.
 
-.. _doc_vulkan_validation_layers-android:
+.. note::
+
+    Enabling validation layers has a performance impact, so only enable them
+    when you actually need the output to debug the application.
+
+Windows/macOS
+-------------
+
+Install the Vulkan SDK `<https://vulkan.lunarg.com/sdk/home>`__, which contains
+validation layers as part of its default installation. No need to enable any
+optional features in the installer; installing the core Vulkan SDK suffices. You
+don't need to reboot after installing the SDK, but you may need to close and
+reopen your current terminal.
```

Comment:
In case of macOS, layers will work only when using Godot built with `use_volk=yes`.